staker-rewards/lib/fraud-detector.js

```diff
@@ -0,0 +1,110 @@
+import clc from "cli-color"
+
+export default class FraudDetector {
+  constructor(
+    context,
+    tokenStaking,
+    bondedECDSAKeepFactory,
+    tbtcSystem,
+    factoryDeploymentBlock
+  ) {
+    this.context = context
+
+    this.tokenStaking = tokenStaking
+    this.bondedECDSAKeepFactory = bondedECDSAKeepFactory
+    this.tbtcSystem = tbtcSystem
+
+    this.factoryDeploymentBlock = factoryDeploymentBlock
+  }
+
+  static async initialize(context) {
+    const {
+      TokenStaking,
+      BondedECDSAKeepFactory,
+      TBTCSystem,
+      factoryDeploymentBlock,
+    } = context.contracts
+
+    const tokenStaking = await TokenStaking.deployed()
+    const bondedECDSAKeepFactory = await BondedECDSAKeepFactory.deployed()
+    const tbtcSystem = await TBTCSystem.deployed()
+
+    return new FraudDetector(
+      context,
+      tokenStaking,
+      bondedECDSAKeepFactory,
+      tbtcSystem,
+      factoryDeploymentBlock
+    )
+  }
+
+  async isOperatorKeepECDSAFraudulent(operator) {
+    // Get all slashing events for the operator.
+    const events = await this.tokenStaking.getPastEvents("TokensSlashed", {
+      fromBlock: this.factoryDeploymentBlock,
+      toBlock: "latest",
+      filter: { operator: operator },
+    })
+
+    if (events.length > 0) {
+      console.log(
```

Comment:
Can we capitalize all log messages in order to align the output with the rest of the script?